client/src/pages/AppView/AppView.js

```diff
@@ -24,14 +24,23 @@ const HeaderSection = ({
     appType,
     logoSrc,
     hasPlugin,
+    organisationSlug,
 }) => (
     <section
         className={classnames(styles.appCardSection, styles.appCardHeader)}
     >
         <AppIcon src={logoSrc} />
         <div>
             <h2 className={styles.appCardName}>{appName}</h2>
-            <span className={styles.appCardDeveloper}>by {appDeveloper}</span>
+            <span className={styles.appCardDeveloper}>
+                by{' '}
+                <a
+                    className={styles.link}
+                    href={`/organisation/${organisationSlug}/view`}
```

Comment:
Is it possible that `organisationSlug` will be null/undefined here leading to a broken link?  Or is it guaranteed to always be populated by the backend?

Reply:
it is guaranteed to not be null/undefined, but  the guarantee is happening at the application layer not the DB, there is no way that it could end up being null/undefined (we use slugify based on the name which is required) unless someone manually alters the DB ..